### ISSUE TITLE
Highlight-lines range is not inclusive

### ISSUE THREAD

**Issue by segeljakt** [Aloxaf/silicon#21]
This code:
```bash
echo "a\nb" | silicon --output foo.png --highlight-lines 1-2 --language python
```
generates an image with only line 1 highlighted, should it not include line 2 as well?

**Comment by KamWithK** [comment]
I'd like to work on fixing this.
One thing I'd like to know at the same time though is whether it is intended for an extra blank line zero to always be inserted into the image?
I believe it should highlight line 2 (assuming you had a -e argument for echo so it had a second line), as line 1 is included.

**Comment by KamWithK** [comment]
I've fixed the issue, it's compiling fine for me, so I don't understand why there is a failed test

**Comment by Aloxaf** [comment]
Closed by #26
